vignettes/stata.Rmd

```diff
@@ -0,0 +1,190 @@
+---
+title: "Comparison with Stata's `margins` command" 
```

Comment:
This will eventually be a "Alternative software packages" page, I think, not just Stata.